### ISSUE TITLE
if using MM/YYYY don't show days

### ISSUE THREAD

**Issue by smcclstocks** [billdami/ember-date-picker#1]
maybe isn't a big deal but it would avoid some confusion when days aren't relevant. Seriously, this could be dumb but just tossing it out there to get your reaction.

**Comment by billdami** [comment]
Yeah, that makes sense. I could also see a use case for hiding the year selector; for instance if you have a birthday field that doesn't require the birth year.

Maybe instead of parsing the `dateFormat` to internally decide what columns/selectors to display, I'll just make it a little more straightforward and declarative, and expose 3 additional optional parameters: `hideMonth`, `hideDay`, and `hideYear`.

**Comment by smcclstocks** [comment]
sounds much better.
